### PR TITLE
USAGOV-29-Initial-Draft: Remove State Dir Config Files

### PR DESCRIPTION
**Pull request by thetypicalprogrammer**
## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-29

## Checklist for the Developer

- [x] A link to the JIRA ticket has been added above.
- [ ] The JIRA ticket identifies the desired result of this change.
- [ ] The JIRA ticket contains clear acceptance criteria.
- [ ] The JIRA ticket contains clear testing steps.
- [ ] The JIRA ticket contains a description of "Done".
- [ ] Any preparation/installation/update steps required for this code to work properly (drush commands, scripts, configuration updates) are documented in the ticket.

## Checklist for the Peer Reviewers
- [ ] The branch name of this PR matches the project standards.
- [ ] QA steps are followed and any changes to process are updated in the ticket.
- [ ] Code Standards are followed, there are no bad practices in use.
- [ ] Config changes (if any) include only necessary changes.
- [ ] No errors in Drupal or Client Browser.
- [ ] Code has been tested locally (if possible).
- [ ] Following AC and Testing Steps verify changes work as expected.
- [ ] There are no known side-effects outside of expected behavior.

### CODE REVIEW

**Comment by thetypicalprogrammer** [comment]
Fix bugs from #354

**Review by cwacht — APPROVED** [review]
The reason I did not approve this before was because when I tested it, all of the styles were missing, and there was another config error related to USWDS Process List. Now, we have identified the cause for the missing styles and Process List error, so I was able to re-test and I am comfortable approving and merging these changes into the dev branch.

**Pull request merged**
